src/patternfly/components/Login/login.scss

```diff
@@ -212,7 +222,7 @@
 .pf-c-login__main-footer-link-text {
   display: none;
```

Comment:
Can we replace `display: none;` with the styles included in the [`.pf-u-sr-only`](https://github.com/patternfly/patternfly-next/blob/b852dc800540e1c27f98b1a0e630d713681492a3/src/patternfly/utilities/Accessibility/accessibility.scss#L5) utility?

Reply:
FYI, I created https://github.com/patternfly/patternfly-next/issues/1146 so we can use an `@include` when we need to include the sr-only or visible class properties.